src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books.map((book) => book.id !== action.payload);
```

Comment:
- [ Required ] I think it would be better if you use the `filter` method in the array prototype to remove a book with a given id, this method allows you to `filter` out items in the array that do not meet the condition that is being tested. I would like to suggest that you refactor this line of code as shown below.

## Example code
```JavaScript
  state.books.filter((book) => book.id !== action.payload);
```